pgjdbc/pom.xml

```diff
@@ -126,6 +126,14 @@
           <outputEncoding>UTF-8</outputEncoding>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>1.6</source>
+          <target>1.6</target>
```

Comment:
Is it really required?

1.8 builds use source=target=1.8 (afaik it should come from parent pom)